E0263 needs to be updated to new format

From: src/test/compile-fail/E0263.rs

E0263 needs a span_label, updating it from:

```
error[E0263]: lifetime name `'a` declared twice in the same scope
  --> src/test/compile-fail/E0263.rs:11:16
   |
11 | fn foo<'a, 'b, 'a>(x: &'a str, y: &'b str) { } //~ ERROR E0263
   |                ^^
```

To:

```
error[E0263]: lifetime name `'a` declared twice in the same scope
  --> src/test/compile-fail/E0263.rs:11:16
   |
11 | fn foo<'a, 'b, 'a>(x: &'a str, y: &'b str) { } //~ ERROR E0263
   |                ^^ declared twice
```

Bonus: underline and label the previous declaration:

```
error[E0263]: lifetime name `'a` declared twice in the same scope
  --> src/test/compile-fail/E0263.rs:11:16
   |
11 | fn foo<'a, 'b, 'a>(x: &'a str, y: &'b str) { } //~ ERROR E0263
   |        --      ^^ declared twice
   |        |
   |        previous declaration here
```

--- a/src/librustc/middle/resolve_lifetime.rs
+++ b/src/librustc/middle/resolve_lifetime.rs
@@ -718,10 +718,14 @@ impl<'a, 'tcx> LifetimeContext<'a, 'tcx> {
                 let lifetime_j = &lifetimes[j];
 
                 if lifetime_i.lifetime.name == lifetime_j.lifetime.name {
-                    span_err!(self.sess, lifetime_j.lifetime.span, E0263,
-                        "lifetime name `{}` declared twice in \
-                                the same scope",
-                                lifetime_j.lifetime.name);
+                    struct_span_err!(self.sess, lifetime_j.lifetime.span, E0263,
+                                     "lifetime name `{}` declared twice in the same scope",
+                                     lifetime_j.lifetime.name)
+                        .span_label(lifetime_j.lifetime.span,
+                                    &format!("declared twice"))
+                        .span_label(lifetime_i.lifetime.span,
+                                   &format!("previous declaration here"))
+                        .emit();
                 }
             }
 

--- a/src/test/compile-fail/E0263.rs
+++ b/src/test/compile-fail/E0263.rs
@@ -8,6 +8,10 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-fn foo<'a, 'b, 'a>(x: &'a str, y: &'b str) { } //~ ERROR E0263
+fn foo<'a, 'b, 'a>(x: &'a str, y: &'b str) {
+    //~^ ERROR E0263
+    //~| NOTE declared twice
+    //~| NOTE previous declaration here
+}
 
 fn main() {}

